.github/workflows/ci-arm64.yml

```diff
@@ -39,12 +51,14 @@ jobs:
         type benchmark.txt
 
   build-openssl-windows-arm64-clangcl:
-    runs-on: [self-hosted, Windows, ARM64, GCC]
+    runs-on: [self-hosted, Windows, ARM64, GCC, D2ps_v5]
```

Comment:
Can you please explain a little bit more why this job is being restricted to that particullar runner size?

Reply:
Yes, it is also option, however less flexible for different hardware tests.